tests/remove_crlf_test.py

```diff
@@ -15,9 +15,35 @@
 )
 def test_remove_crlf(input_s, expected, tmpdir):
     path = tmpdir.join('file.txt')
-    path.write(input_s)
+    input_b = bytes(input_s, 'UTF-8')
```

Comment:
Why those changes?
How is it better to write bytes intead of a unicode string directly?

Reply:
As far as I remember, the write function automatically changed the line endings so there was no real test of the different line endings.